### PR TITLE
Added functionality for 'WITHHASH' in GeoRadiusResponse

### PR DESCRIPTION
**Pull request by Harsh4902**
- Added 'Unsigned52BitInteger' class.
- data field 'hashRawScore' added to hold value when using 'WITHHASH' parameter.
- added a test for this.

solving #3527

### CODE REVIEW

File: src/main/java/redis/clients/jedis/Unsigned52BitInteger.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package redis.clients.jedis;
+
+import java.math.BigInteger;
+
+public class Unsigned52BitInteger {
+  private BigInteger value;
+
+  public Unsigned52BitInteger(Long value) {
+    if (value < 0 || value > 0x1FFFFFFFFFFFFFL) {
```
Comment (sazzad16) [inline src/main/java/redis/clients/jedis/Unsigned52BitInteger.java:9]: This check can be done in test(s).

**Review by sazzad16 — CHANGES_REQUESTED** [review]

File: src/main/java/redis/clients/jedis/Unsigned52BitInteger.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package redis.clients.jedis;
+
+import java.math.BigInteger;
+
+public class Unsigned52BitInteger {
```
[the remark this answers is not in the document]
Comment (Harsh4902) [reply src/main/java/redis/clients/jedis/Unsigned52BitInteger.java:5]: Okay, will remove the 'Unsigned52BitInteger' class.

**Comment by Harsh4902** [comment]
@sazzad16 Please review the last commit.

**Comment by sazzad16** [comment]
@Harsh4902 We already have `rawScore` and won't need `hashRawScore` separately. JavaDoc will be added to clarify the confusion by #3542.

Thank you for investing your precious time in Jedis. Hopefully we'll merge some of your contributions in future.

**Pull request closed without merging**